content/en/blog/_posts/2023-10-18-gateway-api-time-to-switch-i2gw/index.md

```diff
@@ -0,0 +1,111 @@
+
+Subsequently, cluster operators utilize these GatewayClasses to deploy gateways.
+A Gateway in Kubernetes' Gateway-API defines how external traffic can be directed to Services within the cluster, essentially bridging non-Kubernetes sources to Kubernetes-aware destinations.
+It represents a request for a load balancer configuration aligned with a GatewayClass’ specification.
+The Gateway spec may not be exhaustive as some details can be supplied by the GatewayClass controller, ensuring portability.
+Additionally, a Gateway can be linked to multiple Route references to channel specific traffic subsets to designated services.
+
+Lastly, application developers configure route resources (such as HTTPRoutes), to manage configuration (e.g. timeouts, request matching/filter) and Service composition (e.g. path routing to backends)
+Route resources define protocol-specific rules for mapping requests from a Gateway to Kubernetes Services.
+HTTPRoute is for multiplexing HTTP or terminated HTTPS connections.
+It's intended for use in cases where you want to inspect the HTTP stream and use HTTP request data for either routing or modification, for example using HTTP Headers for routing, or modifying them in-flight.
+
+![The resources of Gateway API](gateway-api-resources.svg)
+
+### Portability
+Gateway API is designed to be more portable across different implementations, clusters and environments. It helps reduce Ingress' reliance on non-portable, provider-specific annotations, making your configurations more consistent and easier to manage across multiple clusters.
+
+### Expressiveness
+The Gateway API sets Kuberentes standard support of a wide range of features, such as header-based matching, traffic splitting, weight-based routing, request mirroring and more. With Ingress, these features need custom provider-specific annotations.
+
+### Extensibility
+Gateway API allows for custom resources to be linked at various layers of the API. This makes granular customization possible at the appropriate places within the API structure.
+
+## Getting Started
+
+### Gateway API Quick Start
+
+The first thing you will need is a Kubernetes cluster. You can set up a new cluster using [KinD](https://kind.sigs.k8s.io/) with the following command:
+```bash
+kind create cluster --name introducing-i2gw
+```
+When you create a service of type LoadBalancer in a Kubernetes cluster, typically, cloud providers (like AWS, GCP, Azure) automatically provision a cloud load balancer and allocate a public IP address for it. However, since KinD runs locally, there's no cloud provider available to automatically provision an external IP for such services.
+
+This is where MetalLB comes in. MetalLB provides a network load-balancer implementation for Kubernetes clusters that do not have external load-balancers available, like KinD.
```

Comment:
```suggestion
This is where MetalLB can come in. MetalLB provides a network load-balancer implementation for Kubernetes clusters that do not have external load-balancers available; this could be KinD or it could be your bare-metal production cluster.
```